src/backend/export/outputVendors/googleSheets/googleSheets.ts

```diff
@@ -11,7 +11,11 @@ import { appendToSpreadsheet } from './googleSheetsInternalAPI';
 
 const GOOGLE_SHEETS_DATE_FORMAT = 'DD/MM/YYYY';
 const DEFAULT_SHEET_NAME = '_caspion';
-const COLUMN_HEADERS = ['תאריך', 'סכום', 'תיאור', 'תיאור נוסף', 'קטגוריה', 'מספר חשבון', 'hash - לא לגעת', 'מטבע'];
+const COLUMN_HEADERS = [
+  'תאריך', 'סכום', 'תיאור', 'תיאור נוסף',
+  'קטגוריה', 'מספר חשבון', 'hash - לא לגעת',
+  'מטבע', 'סכום מקורי', 'ת. חיוב', 'מזהה', 'סוג', 'סטטוס'
+];
```

Comment:
```suggestion
const COLUMN_HEADERS = [
  'תאריך',
  'סכום',
  'תיאור',
  'תיאור נוסף',
  'קטגוריה',
  'מספר חשבון',
  'hash - לא לגעת',
  'מטבע',
  'סכום מקורי',
  'ת. חיוב',
  'מזהה',
  'סוג',
  'סטטוס'
];
```
@brafdlog I listed it one-per-line and I think the order is OK. It is a little confusing and I suggest keeping it like that in one-per-line.

@michlus Please accept this suggestion.